docs/pages/blog/mui-x-v8-alpha-zero.md

```diff
@@ -0,0 +1,95 @@
+However, we will only release new v7 versions as needed to patch bugs or add community contributions.
+
+All new features and enhancements will go to v8, and after its first stable release, v7 will officially transition to [long-term support](https://mui.com/x/introduction/support/#long-term-support-lts) status.
+
+## Where's the v8 documentation?
+
+As mentioned above, v7 is still the official version, so by default, the documentation shows v7 features and API.
+The next version's documentation is in the `v8` subdomain.
+
+- [https://next.mui.com/x/](https://next.mui.com/x/)
+
+## What's coming next?
+
+The following is a list of enhancements in the pipeline for v8.
+It contains the highlights we aim to include in the first versions:
+
+### Data Grid
+
+- [Drag and drop with touch support](https://github.com/mui/mui-x/issues/15385)
+- [New Toolbar](https://github.com/mui/mui-x/issues/11584)
+- [Row spanning](https://mui.com/x/react-data-grid/row-spanning/) (without `unstable_` prefix)
+- [Pivoting](https://github.com/mui/mui-x/issues/214) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+### Date and Time Pickers
+
+- [Improved visual customization](https://github.com/mui/mui-x/issues/14753)
+- [Time Range Picker](https://github.com/mui/mui-x/issues/4460) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Support for month and year edition on date range picker](https://github.com/mui/mui-x/issues/4546) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+### Tree View
+
+- [Parent/chidren selection propagation](https://github.com/mui/mui-x/issues/12883)
+- [Lazy loading for tree items](https://github.com/mui/mui-x/issues/9687)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Virtualization](https://github.com/mui/mui-x/issues/9685)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+### Charts
+
+- Improved Design
+- [Radar chart](https://github.com/mui/mui-x/issues/7925)
+- [Improved Data zoom](https://github.com/mui/mui-x/issues/15383)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Funnel chart](https://github.com/mui/mui-x/issues/7929)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+:::info
+You can check our [roadmap](https://github.com/orgs/mui/projects/35) for the full live list.
+:::
+
+## How to migrate?
+
+As usual, we've prepared a [migration guide](https://v8.mui.com/x/migration/migration-data-grid-v7/), and we'll continuously update it as we make any breaking changes during the pre-releases.
+It lists every breaking change you may need to update to migrate your codebase.
+
+We strongly encourage you to try the new version—it builds on the solid foundation we established with v7, bringing even more improvements!
+
+## How to get involved?
```

Comment:
Also not a question—could be "How can I get involved?" but in general I think headers are better framed as action statements

```suggestion
## How to get involved
```